000-debug/0-load.php

```diff
@@ -4,3 +4,9 @@
 
 require_once( __DIR__ . '/logger.php' );
 require_once( __DIR__ . '/debug-mode.php' );
+
+function remove_background_updates_test( $tests ) {
+ unset( $tests['async']['background_updates'] );
+ return $tests;
+}
+add_filter( 'site_status_tests', 'remove_background_updates_test' );
```

Comment:
Let's move this to `001-core.php` and add a comment explaining why we're disabling the test.

Reply:
Moved and added! I guess it was the where the code was located (`000-debug/0-load.php`) that was returning the incorrect success message, not a bug in the code:
<img width="1517" alt="site health tests" src="https://user-images.githubusercontent.com/29221840/77514320-d9854a00-6e33-11ea-9943-abe6443ab722.png">